src/network/adapter.cpp

```diff
@@ -0,0 +1,101 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (C) 2021 YADRO.
+ */
+
+#include "adapter.hpp"
+
+#include "dbus.hpp"
+
+#include <phosphor-logging/log.hpp>
+
+#include <charconv>
+#include <fstream>
+
+using namespace phosphor::logging;
+
+static constexpr const char* pciidsPath = "/usr/share/misc/pci.ids";
+
+const std::string NetworkAdapter::inventoryPath =
+    dbus::inventory::pathBase + std::string("/system/network/adapter/");
+
+const std::pair<std::string, std::string>
```

Comment:
Let's move this to common header

Reply:
Hmm, that has some sense. I agree.

Reply:
But I believe the PCIVendorLookup from `storage/inventory.cpp/hpp` should be removed.
And extra issue: what do you think about renaming `storage/inventory.cpp/hpp` to something `storage/drive.cpp/hpp`?
I guess `inventory` is a too general definition.

Reply:
> But I believe the PCIVendorLookup from storage/inventory.cpp/hpp should be removed.
Right, we can use your implementation.

> And extra issue: what do you think about renaming storage/inventory.cpp/hpp to something storage/drive.cpp/hpp?
My intention was that lately there will be some additional functionality added, like control RAID configuration and manage volumes, so "inventory" here was to show functionality.

Reply:
>  Let's move this to common header

+1

Reply:
FIxed